janitor/math.py

```diff
@@ -124,7 +124,8 @@ def softmax(s: pd.Series) -> pd.Series:
     :param s: Input Series.
     :return: Transformed Series.
     """
-    return scipy_softmax(s)
+
+    return pd.Series(scipy_softmax(s), index=s.index, name=s.name)
```

Comment:
Actually I'd like use `np.exp(s) / sum(np.exp(s))` instead of using scipy

Reply:
I think the SciPy implementation comes with numerical stability checks for free. Is there a reason why you would prefer to use a NumPy implementation?

Reply:
To totally replace scipy. Since we only use it in `math.py` and `functions/impute.py`
- `mode` function in `functions/impute.py` could replace by `Series.value_counts`
- `math.py`
  - `sigmoid` (`expit`) could use `np.exp`
  - `softmax` (`scipy.special.softmax`) could use `np.exp`
  - `logit` (`scipy.special.softmax`) could use `np.log`
  - `normal_cdf` (`scipy.stats.norm.cdf`) and `probit` (`scipy.stats.norm.ppf`) can't replace easily.